### PR TITLE
fix: timeout and rpc-timeout changed to ms

### PR DESCRIPTION
**Pull request by ricochet**
Related to a naming convention change a week ago:
https://github.com/wasmCloud/wash/pull/198

I ran `make && make test` locally.
This regenerated petclinic.rs file. I opted to keep it, but
holler if this needs to be a separate PR.

### CODE REVIEW

File: build/makefiles/actor.mk [hunk 1]
```diff
@@ -91,17 +91,17 @@ start:
...
 inventory:
 	$(WASH) ctl get inventory $(shell $(WASH) ctl get hosts -o json | jq -r ".hosts[0].id")
 
 ifneq ($(wildcard test-options.json),)
 # if this is a test actor, run its start method
 # project makefile can set RPC_TEST_TIMEOUT to override default
-RPC_TEST_TIMEOUT ?= 2
+RPC_TEST_TIMEOUT_MS ?= 2000
 test::
-	$(WASH) call --test --data test-options.json --rpc-timeout $(TEST_TIMEOUT) \
```
Comment (ricochet) [inline build/makefiles/actor.mk:104]: I didn't see other hits for TEST_TIMEOUT, so I assumed this was a bug.

**Review by brooksmtownsend — APPROVED** [review]
Great fix for this breaking change, thank you @ricochet !

File: build/makefiles/actor.mk (hunk 1 above)
Comment (stevelr) [reply build/makefiles/actor.mk:104]: Good catch! Thanks!

File: petclinic/actors/clinicapi/actor.mk [hunk 2]
```diff
@@ -82,18 +82,18 @@ start:
...
 
 ifneq ($(wildcard test-options.json),)
 # if this is a test actor, run its start method
 # project makefile can set RPC_TEST_TIMEOUT to override default
-RPC_TEST_TIMEOUT ?= 2
+RPC_TEST_TIMEOUT_MS ?= 2000
 ACTOR_ID=$(shell make --silent actor_id)
 test::
-	$(WASH) call $(ACTOR_ID) --test --data test-options.json --rpc-timeout $(RPC_TEST_TIMEOUT) Start
+	$(WASH) call $(ACTOR_ID) --test --data test-options.json --rpc-timeout-ms $(RPC_TEST_TIMEOUT) Start
```
Comment (stevelr) [inline petclinic/actors/clinicapi/actor.mk:96]: needs `RPC_TEST_TIMEOUT_MS` here

Comment (ricochet) [reply petclinic/actors/clinicapi/actor.mk]: Thanks for catching this! I updated all of my PR's

**Pull request merged**